public/app/features/correlations/Forms/CorrelationFormNavigation.tsx

```diff
@@ -12,21 +13,23 @@ export const CorrelationFormNavigation = () => {
 
   const LastPageNext = !readOnly && (
     <Button variant="primary" icon={loading ? 'fa fa-spinner' : 'save'} type="submit" disabled={loading}>
-      {correlation === undefined ? 'Add' : 'Save'}
+      {correlation === undefined
+        ? t('correlations.nav-form.add-btn', 'Add')
```

Comment:
I think it'd be better to use full words instead of abbreviation (same for other keys, like `qe` for `query editor`)

Reply:
here, should I expand 'nav' or 'btn' or both?

Reply:
Also, I now extracted the map code into a variable but not sure how to access it in UI. Let me know if it is working correctly

Reply:
The variable map works great 🙌  

I think it should be `-button`, not `-btn` to keep it consistent with other keys.